cartridges/int_klaviyo_sfra/cartridge/controllers/Order.js

```diff
@@ -29,6 +30,29 @@ server.append('Confirm', function (req, res, next) {
                     dataObj = orderConfirmationData.getData(currentOrder, exchangeID);
                     klaviyoUtils.trackEvent(exchangeID, dataObj, klaviyoUtils.EVENT_NAMES.orderConfirmation, currentOrder.customerEmail);
                 }
+
```

Comment:
Ah okay I see you made the changes I mentioned above in the SG controller here, this looks good. 1 thing is I believe Devin mentioned we won't need to pass along the session vars to `subscribeUser` because we can access them globally

Reply:
fair point, fixing